cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -1789,6 +1789,15 @@ public CloudBuild getBuild(UUID buildGuid) {
         return resourceMapper.mapResource(responseEntity.getBody(), CloudBuild.class);
     }
 
+    @Override
+    public List<CloudBuild> getAllBuildsForApplication(UUID applicationGuid) {
+        Map<String, Object> urlVars = new HashMap<>();
+        urlVars.put("applicationGuid", applicationGuid);
+        List<CloudBuild> cloudBuilds = doGetResources("/v3/apps/{applicationGuid}/builds", urlVars, CloudBuild.class);
```

Comment:
Sonar issue: Return the result directly instead of assigning it to a variable first.

Reply:
Done